addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -125,6 +125,18 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 
     .o_pc_overlay {
         @include o-position-absolute(0,0,0,0);
+        // Allow hover effect on svg paths in order to display bootstrap tooltip,
+        // by pushing the absolute div behind it
+        z-index: -1;
+
+        // Only display single line for the next rank, and display ellipsis if it can not fit in the card
+        .o_pc_text_overflow {
+            @include o-text-overflow(-webkit-box);
```

Comment:
Did not see a single occurrence of `@include o-text-overflow(-webkit-box);` so it makes me a bit suspicious.

Is the vendor prefix required here?

We tend to avoid those.

Reply:
See https://github.com/odoo/odoo/pull/92871/files#r949843249 to see from where the reference was taken and why. Basically removing vendor prefix breaks edition.

Reply:
@dja-odoo 

Just checked out locally and gave it a try, something like this seems to work:

```scss
        h4 {
            width: 60%;

            // only apply ellipsis when in non-edit mode, that way the editor can see the full value
            &:not(.o_editable) {
                overflow: hidden;
                text-overflow: ellipsis;
                white-space: nowrap;
            }
        }
```

For both edition and non-edition mode.
I would really like to avoid vendor prefixes if possible and use standard stuff.
This is just some text we want to crop, there really should not be a need to introduce new rules that are not used anywhere else.

Reply:
Hello @awa-odoo, in edition mode it will look like this for the long string
![Screenshot 2022-09-27 at 15 49 38](https://user-images.githubusercontent.com/7599908/192501677-30b64077-7aac-4494-8f6a-86c29050da72.png)
If that's okay, we can indeed avoid vendor prefix :)

Reply:
That's fine, it's a very side case, I'd rather not make the code too complex :)

Reply:
@awa-odoo Thanks for the quick reply, done :)